### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ispirett**
Thanks for your Time

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+}
+.body-image-text {
+  position: relative;
+  top: -29em;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.image-text {
```
Comment (JamesOkunlade) [inline css/main.css]: Remove empty selector!!

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+  margin: -1.6rem auto;
+}
+
+/* footer */
+
+.footer {
+  border-top: 2px solid rgb(221, 221, 221);
+  border-bottom: 2px solid rgb(221, 221, 221);
+  margin: 65rem auto 0 auto;
+
```
Comment (JamesOkunlade) [inline css/main.css:384]: Remove whitespace. Always your code well indented and free of unneeded whitespace.

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+}
+
+.footer-body {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.footer-section {
```
Comment (JamesOkunlade) [inline css/main.css]: Remove!!

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,614 @@
...
+
+.footer-sidebar ul {
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+.footer-sidebar h3 {
+  font-size: 14px;
+  padding-left: 10px;
+
```
Comment (JamesOkunlade) [inline css/main.css]: Here too. whitespace.

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,614 @@
...
+}
+
+.css-r5ic95 {
+  width: 10px;
+  padding-right: 5px;
+}
+
+.footer-nav {
+  text-align: center;
+
```
Comment (JamesOkunlade) [inline css/main.css]: !!!!

File: css/main.css [hunk 6]
```diff
@@ -0,0 +1,614 @@
...
+  margin: 0 auto;
+  width: 80%;
+}
+.footer-nav ul {
+}
+.footer-nav ul li {
+  display: inline-block;
+
```
Comment (JamesOkunlade) [inline css/main.css]: !!!

**Review by JamesOkunlade — APPROVED** [review]
### Great work on newyorktimes page guys!!!

Looks good to me. Refactor your code according to my different comments on different lines in your CSS.

Position your logo to the center.

Merge development to master, close PR and fill the completion form.

Here are your thumbs up :+1::+1: Goodluck in your future projects.

**Comment by SotoArmando** [comment]
Thanks n.n

**Comment by Ispirett** [comment]
Thanks for your review we will

**Pull request merged**